From ca98169d9a2bf9d9a0fe52fb4f26035ca71481d5 Mon Sep 17 00:00:00 2001
From: ryanheinen <ryan.heinen@gmail.com>
Date: Sat, 14 Sep 2024 22:43:54 -0700
Subject: [PATCH 07/27] Remove spaces from layer name to see if that fixes
 build problem

---
 miryoku/custom_config.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/miryoku/custom_config.h b/miryoku/custom_config.h
index 5436cc8..ea12c91 100644
--- a/miryoku/custom_config.h
+++ b/miryoku/custom_config.h
@@ -56,7 +56,7 @@ U_NP,              U_NP,              U_LT(U_MEDIA, ESC), U_LT(U_NAV, RET),  U_L
 U_BOOT,            &u_to_U_TAP,       &u_to_U_EXTRA,     &u_to_U_BASE,      U_NA,              &kp LBKT,          &kp N7,            &kp N8,            &kp N9,            &kp RBKT,          \
 &kp LGUI,          &kp LALT,          &kp LCTRL,         &kp LSHFT,         U_NA,              &kp EQUAL,         &kp N4,            &kp N5,            &kp N6,            &kp N0,          \
 U_NA,              &kp RALT,          &u_to_U_NAV,       &u_to_U_NUM,       U_NA,              &kp BSLH,          &kp N1,            &kp N2,            &kp N3,            &kp GRAVE,         \
-U_NP,              U_NP,              U_NA,              U_NA,              U_NA,              U_NA,              U_NA,              U_NA,              U_NP,              U_NP
+U_NP,              U_NP,              U_NA,              U_NA,              U_NA,              U_NA,               U_NA,              U_NA,              U_NP,              U_NP
 
 // copy of MIRYOKU_ALTERNATIVES_FUN_FLIP with right thumbs replaced with to NA
 #define MIRYOKU_LAYER_FUN_FLIP \
@@ -83,8 +83,8 @@ MIRYOKU_X(MEDIA,  "Media") \
 MIRYOKU_X(NUM,    "Num") \
 MIRYOKU_X(SYM,    "Sym") \
 MIRYOKU_X(FUN,    "Fun") \
-MIRYOKU_X(NUM_FLIP,    "Num Flip") \
-MIRYOKU_X(FUN_FLIP,    "Fun Flip")
+MIRYOKU_X(NUM_FLIP,    "NumFlip") \
+MIRYOKU_X(FUN_FLIP,    "FunFlip")
 
 #define MIRYOKU_LAYERMAPPING_NUM_FLIP MIRYOKU_MAPPING
 #define MIRYOKU_LAYERMAPPING_FUN_FLIP MIRYOKU_MAPPING
-- 
2.52.0

