From e3b2ce184afb3fe3fd3cc5ce7aae02d668a98d67 Mon Sep 17 00:00:00 2001
From: ryanheinen <ryan.heinen@gmail.com>
Date: Wed, 18 Sep 2024 20:34:16 -0700
Subject: [PATCH 18/27] . to DOT, / to SLASH

---
 miryoku/miryoku_clipboard.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/miryoku/miryoku_clipboard.h b/miryoku/miryoku_clipboard.h
index 7993a05..317b23d 100644
--- a/miryoku/miryoku_clipboard.h
+++ b/miryoku/miryoku_clipboard.h
@@ -17,10 +17,10 @@
   #define U_UND &kp LG(Z)
 #elif defined (MIRYOKU_CLIPBOARD_WIN)
   #define U_RDO &kp LC(T)
-  #define U_PST &kp LC(.)
+  #define U_PST &kp LC(DOT)
   #define U_CPY &kp LC(I)
   #define U_CUT &kp LC(B)
-  #define U_UND &kp LC(/)
+  #define U_UND &kp LC(SLASH)
 #else
   #define U_RDO &kp K_AGAIN
   #define U_PST &kp LS(INS)
-- 
2.52.0

