From c22b4bac2bf95ee9c5212d841afa383831700688 Mon Sep 17 00:00:00 2001
From: ryanheinen <ryan.heinen@gmail.com>
Date: Wed, 23 Oct 2024 17:40:15 -0700
Subject: [PATCH 25/27] Revert back to default clipboard

---
 miryoku/miryoku_clipboard.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/miryoku/miryoku_clipboard.h b/miryoku/miryoku_clipboard.h
index 317b23d..de932ad 100644
--- a/miryoku/miryoku_clipboard.h
+++ b/miryoku/miryoku_clipboard.h
@@ -16,11 +16,11 @@
   #define U_CUT &kp LG(X)
   #define U_UND &kp LG(Z)
 #elif defined (MIRYOKU_CLIPBOARD_WIN)
-  #define U_RDO &kp LC(T)
-  #define U_PST &kp LC(DOT)
-  #define U_CPY &kp LC(I)
-  #define U_CUT &kp LC(B)
-  #define U_UND &kp LC(SLASH)
+  #define U_RDO &kp LC(Y)
+  #define U_PST &kp LC(V)
+  #define U_CPY &kp LC(C)
+  #define U_CUT &kp LC(X)
+  #define U_UND &kp LC(Z)
 #else
   #define U_RDO &kp K_AGAIN
   #define U_PST &kp LS(INS)
-- 
2.52.0

